### DISCUSSION TITLE
Small part

### DISCUSSION THREAD

**Discussion by 5axes** [5axes/UniversalCuraSettings#18] [Profile or Intent]
Settings for small parts.

Issues with bed adhesion 
Reduce acceleration and jerk ?

Example part of that we can qualify of "Small part" 
![image](https://user-images.githubusercontent.com/11015345/116818253-b49cf380-ab6a-11eb-91d9-9064f79ac870.png)

**Comment by printingotb** [comment]
That's something I haven't tested much. Does lower jerk help or worsen it. Low means it stays longer in the corner wich could improve adhesion. However it could also mean that it sticks more to the nozzle? May depend on the bed material.

Higher nozzle temp on initial layer can make a big improvement to bed adhesion.

**Reply by 5axes** [reply]
Well in this case the actual solution I'm currently testing is using a Raft. It solved the bed adhesion issue . But the main problem is how to fix the "Raft Air Gap"  0.3 as default in fdmprinter.def.json

**Reply by 5axes** [reply]
My test part for small part, ![20210502_161807](https://user-images.githubusercontent.com/11015345/116816512-81eefd00-ab62-11eb-87bf-85e0a7ec57bc.jpg)

Combine tests with usefulness. by producing customizable nail clamp : https://www.thingiverse.com/thing:4816588

**Reply by printingotb** [reply]
Why not use a brim?

**Reply by 5axes** [reply]
Because it is much easier to remove and if the part has a fairly complex profile with small details the brim  may be difficult to print. 
![20210502_170331](https://user-images.githubusercontent.com/11015345/116817815-92a27180-ab68-11eb-8bcd-025a314d9b0e.jpg)

**Reply by 5axes** [reply]
The Arachne release by creating  much more simple paths on small part  are also much more easy to remove from the raft.

https://user-images.githubusercontent.com/11015345/116820565-1ca50700-ab76-11eb-8e77-c611f9ec2c57.mov

**Reply by printingotb** [reply]
That's nice!
